tuxemon/locale.py

```diff
@@ -65,6 +65,11 @@ class LocaleInfo:
     domain: str
     path: str
 
+class MoneyInfo:
+    @staticmethod
+    def get_current_money(session: Session) -> str:
```

Comment:
I still don't think this needs to be a class, and we don't really need this function anyway... Let's just move the code inline.